### PR TITLE
ugarit: init at 2.0

### PR DESCRIPTION
**Pull request by ebzzry**

### CODE REVIEW

File: pkgs/tools/backup/ugarit/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{ pkgs, stdenv, eggDerivation, fetchegg }:
+let
+  eggs = import ./eggs.nix { inherit pkgs stdenv; };
+in with pkgs; eggDerivation rec {
+  pname = "ugarit";
+  version = "2.0";
+  name = "${pname}-${version}";
+
+  src = fetchegg {
+    name = "${pname}";
```
Comment (zimbatm) [inline pkgs/tools/backup/ugarit/default.nix]: you can use `name = pname;`

File: pkgs/tools/backup/ugarit/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    version = "${version}";
```
Comment (zimbatm) [inline pkgs/tools/backup/ugarit/default.nix]: you can use `inherit version;`

**Comment by ebzzry** [comment]
@zimbatm done =)

**Comment by zimbatm** [comment]
Thanks !

**Comment by ebzzry** [comment]
Thanks! =D

**Comment by ebzzry** [comment]
By the way, I did the same with ugarit-manifest-maker (#13089)

**Pull request merged**